src/box/replication.cc

```diff
@@ -930,13 +942,33 @@ struct replicaset_connect_state {
 	int failed;
 	/** Signaled when an applier connects or stops. */
 	struct fiber_cond wakeup;
+	/** An applier which connected to the bootstrap leader. */
+	struct applier *bootstrap_leader_applier;
 };
 
 struct applier_on_connect {
 	struct trigger base;
 	struct replicaset_connect_state *state;
 };
 
+struct tt_uuid cfg_bootstrap_leader_uuid;
+struct uri cfg_bootstrap_leader_uri;
```

Comment:
Don't we keep all the globals in the beginning of `replication.cc`?

Reply:
Ok, moved the definitions closer to the beginning of the file:
```diff
diff --git a/src/box/replication.cc b/src/box/replication.cc
index 6487e785d..2310430c1 100644
--- a/src/box/replication.cc
+++ b/src/box/replication.cc
@@ -58,6 +58,9 @@ bool replication_skip_conflict = false;
 bool replication_anon = false;
 int replication_threads = 1;
 
+struct tt_uuid cfg_bootstrap_leader_uuid;
+struct uri cfg_bootstrap_leader_uri;
+
 struct replicaset replicaset;
 
 struct rlist replicaset_on_quorum_gain =
@@ -880,9 +883,6 @@ struct applier_on_connect {
        struct replicaset_connect_state *state;
 };
 
-struct tt_uuid cfg_bootstrap_leader_uuid;
-struct uri cfg_bootstrap_leader_uri;
-
 /** Check if the applier is connected to the configured bootstrap leader. */
 static bool
 is_applier_cfg_bootstrap_leader(const struct applier *applier)
```